tests/test_nonbonded_interaction_group.py

```diff
@@ -0,0 +1,129 @@
+import jax
+
+jax.config.update("jax_enable_x64", True)
+
+import numpy as np
+import pytest
+from common import GradientTest, prepare_reference_nonbonded
+from simtk.openmm import app
+
+from timemachine.fe.utils import to_md_units
+from timemachine.ff.handlers import openmm_deserializer
+from timemachine.lib import potentials
+from timemachine.lib.potentials import NonbondedInteractionGroup
+
+
+@pytest.fixture(autouse=True)
+def set_random_seed():
+    np.random.seed(2021)
+    yield
+
+
+@pytest.fixture
+def test_system():
```

Comment:
Pytest aside: I didn't realize it was okay for fixture names to start with `test_`!

I don't see a prohibition in pytest docs, and confirmed `test_system`, `test_conf`, `test_box` aren't spuriously collected by default...

But it might still be safer if these are renamed to not start with `test_`?

Reply:
Agreed! Fixed in ecf76d8